docs/user_guide/assessment.rst

```diff
@@ -397,6 +397,61 @@ see the :ref:`sphx_glr_auto_examples_plot_new_metrics.py` notebook in the
 :ref:`examples`.
 
 
+Plotting with MetricFrame
+-------------------------
+MetricFrame pd.DataFrame properties can be plotted using matplotlib. Here is an example,
+referencing the same `metric_c_f` object above.
+
+.. code-block::
+
+    metric_c_f.by_group.plot(kind="bar",
+                             title='Show accuracy_score metric')
+
+It is possible to customize the plots. Here are some common examples:
+
+Customize Plots: `ylim`
+^^^^^^^^^^^^^^^^^^^^^^^
+The yaxis range is automatically set, which sometimes can be misleading, hence it is
```

Comment:
A few small suggestions:
* yaxis -> y-axis
* ... which sometimes can be misleading, hence it is ... -> which can be misleading. Therefore it is